src/app/kbv/controllers/load-question.js

```diff
@@ -26,12 +28,8 @@ class LoadQuestionController extends BaseController {
       next();
     });
   }
-  next(req) {
-    if (req.session?.question) {
-      return "question";
-    }
-
-    return "done";
+  hasQuestion(req) {
+    return req.session?.question !== undefined;
```

Comment:
```suggestion
    return !!req.session?.question;
```
I still prefer the `!!` syntax for conciseness and clarity, we can chalk it up to personal preference.